CHANGELOG.rst

```diff
@@ -12,6 +12,11 @@ Changed
 3.3.0 - October 06, 2020
 ------------------------
 
+Added
+~~~~~
+* Added st2-auth-ldap pip requirements for Ldap integartion. (new feature) #5082
```

Comment:
The Changelog record needs to be placed under the `in development` section, instead of already released `3.3.0`. Should be easy to fix it in this PR.

Reply:
Updated the Changelog